server/@types/manage-recalls-api/models/RecallLite.ts

```diff
@@ -0,0 +1,25 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import {RecallResponse} from "./RecallResponse";
+
+export type RecallLite = {
+    assigneeUserName?: string;
+    createdByUserId: string;
+    createdDateTime: string;
+    dossierEmailSentDate?: string;
+    dossierTargetDate?: string;
+    firstName: string;
+    inCustodyAtAssessment?: boolean;
+    inCustodyAtBooking?: boolean;
+    lastName: string;
+    lastUpdatedDateTime: string;
```

Comment:
Optional property?

Reply:
as discussed, not option as sent as same value as createdDateTime initially